fw-child/resources/scss/_template.scss

```diff
@@ -340,10 +355,60 @@
     padding: 0;
   }
 
+  & > ul {
+    padding-top: .75rem;
+  }
+
+  li {
+    margin-bottom: .75rem;
+
+    li:first-child {
+      padding-top: .75rem;
+    }
+
+    li:last-child {
+      margin-bottom: 0;
+    }
+  }
+
   a {
+    color: $gray-900;
     display: inline-block;
     font-size: 1.125rem;
-    padding: .125rem 0 .125rem .5rem;
+    padding: .125rem .125rem .125rem .5rem;
     width: 100%;
+    line-height: 1.25;
+
+    &:hover {
+      text-decoration: none;
+    }
+  }
+}
+
+//
+// ABOUT US SOCIAL NETWORKS
+//
+.social-networks {
```

Comment:
I removed the border and the custom margin and padding as it may not apply in every case and applied it in the builder with bootstrap classes.

I think the rest of the styles should stay there as a base style for the social networks. This way, if we add them anywhere else, there will be a minimum and it will be working out of the box. It's always possible to add some more specific styles later by creating a class that will override these.